components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -442,6 +445,42 @@ export function CreateWorkspacePage() {
     );
 }
 
+function LocalSSHWarning(props: { className: string }) {
+    const isLocalSSHProxyEnabled = useFeatureFlag("gitpod_desktop_use_local_ssh_proxy");
+
+    if (!isLocalSSHProxyEnabled) {
+        return null;
+    }
+
+    return (
+        <div className={props.className}>
+            <Alert light className="bg-gray-100 dark:bg-gray-800" type="info">
+                To make VS Code Desktop open seamlessly we will add a single entry to your local SSH configuration.
+                Gitpod does not read any of your existing SSH configurations.&nbsp;
```

Comment:
**thought**: Re-posting the copy suggestion from https://github.com/gitpod-io/gitpod/pull/18112#discussion_r1252185566 in case there are any bits that come across nicer or less scary. Feel free to update or leave the copy as is.

```suggestion
                Choosing VS Code for Desktop will add a single entry to your local SSH configuration.
                Gitpod won't interfere with your existing SSH configurations.&nbsp;
```